src/Silex/Provider/TwigServiceProvider.php

```diff
@@ -79,7 +80,9 @@ public function register(Application $app)
                 }
 
                 if (isset($app['fragment.handler'])) {
-                    $app['fragment.renderer.hinclude']->setTemplating($twig);
+                    if (method_exists(HIncludeFragmentRenderer::class, 'setTemplating')) {
```

Comment:
same question as before, what version is this in?

Reply:
It exists only in old Symfony versions and was removed in v5